public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -865,7 +865,7 @@ export class LokiDatasource
           : {
```

Comment:
@grafana/observability-logs Do we need to explicitly talk about labels in the success message?

Reply:
I don't think so. I think it is enough to talk about labels only if we won't receive them. So I would be okay with removing that part.